src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -202,12 +202,11 @@ public Result Complete()
 
         public bool DeleteOrderItem(CatalogueItemId catalogueItemId)
         {
-            var orderItem = orderItems.SingleOrDefault(x => x.CatalogueItem.Id == catalogueItemId);
+            var orderItem = orderItems.SingleOrDefault(o => o.CatalogueItem.Id == catalogueItemId);
             if (orderItem == null)
                 return false;
```

Comment:
I believe `List<T>.Remove` accepts `null` so we should be able to omit the `if` statement but please correct me if I'm wrong.

Reply:
done with list remove